apps/aecore/test/aecore_sync_SUITE.erl

```diff
@@ -220,19 +238,33 @@ tx_first_pays_second(Config) ->
     true = ensure_new_tx(N2, NewTx).
 
 ensure_tx_pools_empty(Config) ->
+    ensure_tx_pools_n_txs_(Config, 0).
+
+ensure_tx_pools_one_tx(Config) ->
+    ensure_tx_pools_n_txs_(Config, 1).
+
+ensure_tx_pools_n_txs_(Config, TxsCount) ->
     Ns = [N || {_,N} <- ?config(nodes, Config)],
     retry(
       fun() ->
-              Results = lists:map(
-                          fun(N) ->
-                                  {ok, Pool} = get_pool(N),
-                                  ct:log("Pool (~p) = ~p", [N, Pool]),
-                                  Pool
-                          end, Ns),
-              lists:all(fun([]) -> true;
-                           (_)  -> false
-                        end, Results)
-      end, {?LINE, ensure_tx_pools_empty, Ns}).
+              [APool | RestPools] =
+                  lists:map(
+                    fun(N) ->
+                            case get_pool(N) of
+                                {ok, Pool} when is_list(Pool) ->
+                                    ct:log("Pool (~p) = ~p", [N, Pool]),
+                                    Pool
+                            end
+                    end, Ns),
+              if
+                  length(APool) =:= TxsCount ->
+                      lists:all(fun(P) when P =:= APool -> true;
```

Comment:
This will work since the pools are consistently ordered, but I'm not sure if this fact is that well documented.

Reply:
> This will work since the pools are consistently ordered, but I'm not sure if this fact is that well documented.

Tracked this documentation and test as https://www.pivotaltracker.com/story/show/154362443